testdata/ce/go.tpl

```diff
@@ -0,0 +1,67 @@
+{{- $suffix := randAlphaNum 5 | lower -}}
+apiVersion: codeengine.cloud.ibm.com/v1beta1
+kind: JobRun
+metadata:
+  name: {{ .Name }}-{{ $suffix }}
+spec:
+  jobDefinitionSpec:
+    template:
+      containers:
+      - name: iter8
+        image: puffinmuffin/iter8
+        command:
+        - "/bin/sh"
+        - "-c"
+        - |
+          set -e
+          # trap 'kill $(jobs -p)' EXIT
+
+          # get experiment from secret
+          kubectl get secret {{ .Name }}-{{ $suffix }} -o go-template='{{"{{"}} .data.experiment {{"}}"}}' | base64 -d > experiment.yaml
+
+          # local run
+          export LOG_LEVEL=info
+          iter8 run experiment.yaml
+
+          # update the secret
+          kubectl create secret generic {{ .Name }}-{{ $suffix }} --from-file=experiment=experiment.yaml --dry-run=client -o yaml | kubectl apply -f -
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Name }}-{{ $suffix }}
+stringData:
+  experiment: |
+    # task 1: generate HTTP requests for https://example.com
+    # collect Iter8's built-in latency and error-related metrics
+    - task: gen-load-and-collect-metrics
+      with:
+        versionInfo:
+        - url: https://iter8-ce-demo.fgw1ut94lpp.ca-tor.codeengine.appdomain.cloud
```

Comment:
Should the URL value be replaced with something like ...

```
{{ .CEAppName }}-{{ .CEAppNamespace }}.ca-tor.codeengine.appdomain.cloud
```

Reply:
I wonder how useful templating this will be because the URL also contains the location `ca-tor`, right?

Reply:
What  is this `ca-tor`? Also, does it make sense the whole URL field?

Reply:
```
- url: {{ .URL }}
```

Reply:
`ca-tor` is the cluster zone. It's Toronto, Canada. But yes, I think it makes more sense to template the entire URL field.